src/main/java/no/nav/familie/integrasjoner/config/CacheConfig.kt

```diff
@@ -0,0 +1,29 @@
+package no.nav.familie.integrasjoner.config
+
+import com.github.benmanes.caffeine.cache.Caffeine
+import org.springframework.cache.Cache
+import org.springframework.cache.CacheManager
+import org.springframework.cache.annotation.EnableCaching
+import org.springframework.cache.concurrent.ConcurrentMapCache
+import org.springframework.cache.concurrent.ConcurrentMapCacheManager
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import java.util.concurrent.TimeUnit
+
+@Configuration
+@EnableCaching
+class CacheConfig {
+
+    @Bean
+    fun cacheManager(): CacheManager = object : ConcurrentMapCacheManager() {
+        override fun createConcurrentMapCache(name: String): Cache {
+            val concurrentMap = Caffeine
+                    .newBuilder()
+                    .initialCapacity(100)
+                    .maximumSize(250)
```

Comment:
Var denne 1000 før? Isåfall kan den kanskje endrest til 1000 her også?

Reply:
Tja, de hadde en `ResourcePool` som de delte, jag tenkte att de delte antall plasser då. Men laget en test nå og de hade 1000 per cache. Fikser